api/libproject/devstart.rb

```diff
@@ -65,6 +66,13 @@ def dev_up(*args)
   })
 end
 
+def run_tests(*args)
```

Comment:
At some point we may want to allow for passing an arg that turns on verbose test logging.

(But you can always run tests with that directly using Gradle. So this works.)

Reply:
You can do this:
```shell
./project.rb test --tests org.pmiops.workbench.google.DirectoryServiceImplTest --project-prop verboseTestLogging=yes
```

Reply:
Good call, didn't see args below. Thanks!